views_euid/privacy.hbs

```diff
@@ -6,137 +6,159 @@
       <li>Where an advertiser asks us to create an EUID, at the advertiser’s request, we will share the EUID with service providers who store or process the data for the advertiser.</li>
     </ul>
     <p>We may also disclose personal information in response to lawful requests from public authorities, including to meet security or law enforcement requirements.</p>
+
     <h2>International transfers</h2>
-    <p>When we transform the email address or phone number into an EUID, we do this in Europe. If an EUID participant from outside the EEA and the UK provides us with an email address or phone number, so that we can create an EUID, then we will share the EUID with that EUID participant; this will include (but is not limited to) partners in the US. The Trade Desk itself will also process the EUID in the US.</p>
-    <p>Whenever we transfer personal data outside the EEA and the UK, we ensure that there is a valid transfer mechanism, primarily by using data transfer agreements which are valid in the EEA and the UK.</p>
+    <p>When we transform the email address or phone number into an EUID, we do this in the UK, EEA or Switzerland. If an EUID participant from outside the UK EEA, and Switzerland provides us with an email address or phone number, so that we can create an EUID, then we will share the EUID with that EUID participant; this will include partners in the US (but it could also be a partner located in any country in the world).</p>
+    <p>We transfer personal data outside the UK, EEA and Switzerland under valid transfer mechanisms:</p>
+    <ul class="spaced">
+      <li>The Trade Desk has certified its participation in the EU-U.S. and Swiss-U.S. Data Privacy Frameworks (DPF) and the UK Extension to the EU-U.S. DPF as set forth by the U.S. Department of Commerce, and is committed to adhering to DPF principles with regard to the processing of personal data received from the EU and UK in reliance of DPF. We currently do not rely on the Swiss-U.S. DPF to transfer personal information to the U.S. until Switzerland’s recognition of adequacy, however, The Trade Desk is committed to adhering to Swiss-U.S. DPF Principles with regard to the processing of personal data received in reliance on the Swiss-U.S. DPF. The Trade Desk is responsible for ensuring that onward transfers to third parties are in accordance with the EU-U.S. DPF, the UK Extension to the EU-U.S. DPF and the Swiss-U.S. DPF. To learn more about the DPF program, and to view our certification, please visit https://www.dataprivacyframework.gov/.</li>
+      <li>If deemed necessary under applicable data protection legislation, the standard contractual clauses for data transfers, as approved by the European Commission Implementing Decision (EU) 2021/914 of 4 June 2021 for the transfer of Personal Data to third countries.</li>
```

Comment:
Should the data privacy link above be in href with a link and "here" like the other links in this PR?

Reply:
I just did what I was told:

![image](https://github.com/user-attachments/assets/40f78ac6-3c3e-4a0e-99f4-07412d50df8b)